src/olympia/reviewers/views.py

```diff
@@ -1386,6 +1400,27 @@ def retrieve(self, request, *args, **kwargs):
         )
         return Response(serializer.data)
 
+    @action(
+        detail=True,
+        methods=['get', 'put', 'delete'],
+        permission_classes=ReviewAddonVersionMixin.permission_classes)
+    def draft_comment(self, request, **kwargs):
+        version = self.get_object()
+
+        if request.method == 'GET':
+            instance = get_object_or_404(DraftComment, version=version)
+            serializer = DraftCommentSerializer(instance, data=request.data)
+            return Response(serializer.data)
+        elif request.method == 'DELETE':
+            DraftComment.objects.filter(version=version).delete()
+            return Response(status=status.HTTP_202_ACCEPTED)
+
+        instance, _ = DraftComment.objects.get_or_create(version=version)
```

Comment:
I know the method is limited to `GET`, `DELETE` and `PUT` but I'd be happier if this was also in a `elif request.method == 'PUT`:` block.

Reply:
Fixed in 2047d45